spec/market_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+RSpec.describe Market do 
+    before(:each) do
+        @market = Market.new("South Pearl Street Farmers Market")
+        @vendor1 = Vendor.new("Rocky Mountain Fresh")
+        @vendor2 = Vendor.new("Ba-Nom-a-Nom") 
+        @vendor3 = Vendor.new("Palisade Peach Shack")    
+        @item1 = Item.new({name: 'Peach', price: "$0.75"})
+        @item2 = Item.new({name: 'Tomato', price: '$0.50'})
+        @item3 = Item.new({name: "Peach-Raspberry Nice Cream", price: "$5.30"})
+        @item4 = Item.new({name: "Banana Nice Cream", price: "$4.25"})
+    end
+
+    it '#exists' do 
+        expect(@market).to be_an_instance_of(Market)
+    end
+
+    it '#has a name' do
+        expect(@market.name).to eq ("South Pearl Street Farmers Market")
+    end
+
+    it '#has no vendors' do
+        expect(@market.vendors).to eq([])
+    end
```

Comment:
Functionality that happens in the instantiation can be consolidated into a single test, if you'd like to save time and lines.